ase/scripts/generate_ase_environment.py

```diff
@@ -238,12 +240,16 @@ def config_sources(fd, filelist):
             if (sl[-5:] == '.json'):
                 json_srcs.append(s)
 
+    qsys_sim_files = config_qsys_sources(filelist, vlog_srcs)
```

Comment:
Was Qsys from Quartus installation? Did it use the old version files in ASE simulation previously?

Reply:
Qsys is part of Quartus.

I don't understand the 2nd question.

Reply:
I just wonder how it worked before this change. ASE simulation did not compile Qsys before?

Reply:
There was code in setup scripts just for Qsys in the internal intel-fpga-afu repository. Everything else has been moved out of that script and into afu_sim_setup/generate_ase_environment. This is the last piece.

There's another pull request on that repository, removing the code there. The tests were done with the two changes combined.

Reply:
Are Qsys files needed for ASE simulation? Or it is only used for RTL sysnthesis?

Reply:
Simulation.